app/experimenter/static/js/components/DesignForm.js

```diff
@@ -28,16 +29,21 @@ class DesignForm extends React.PureComponent {
     };
   }
 
+  isMultiPref() {
+    return this.state.data.get("is_multi_pref", this.props.isMultiPref);
```

Comment:
Does this do the right thing.....?  Maybe it does....

Reply:
i think it does?